test/Protocol.js

```diff
@@ -251,7 +251,7 @@ contract('OpenST', function(accounts) {
 		    it("register utility token on value chain", async() => {
 
 		     	const result = await registrarVC.registerUtilityToken(openSTValue.address, symbol, name, conversionRate, conversionRateDecimals,
-		    		CHAINID_UTILITY, requester, registeredBrandedTokenUuid, { from: intercommVC });
+		    		CHAINID_UTILITY, stakerVC, registeredBrandedTokenUuid, { from: intercommVC });
```

Comment:
this is not necessary I think, but does not harm

Reply:
+1